assets/large_language_models/prompts/chat_user_questions/prompt/prompt.prompty

```diff
@@ -0,0 +1,64 @@
+---
+name: Generate user questions on a product
+description: >
+  To submit your application to evaluation, you can ask
+  a language model to generate synthetic questions for you.
+  Use this prompt as a basis to generate a set of questions
+  on a product for a given context and user persona.
+  Using this technique, you can create an evaluation
+  dataset, to measure the quality of your application
+  responses on more than just a few manual samples.
+
+model:
+  api: chat
+  parameters:
+    temperature: 0.8
+
+inputs:
+  persona:
+    type: string
+    description: The user persona
+  context:
+    type: string
+    description: The context where the user is accessing the product
+  product:
+    type: string
+    description: The product the user is interested in
+  count:
+    type: integer
+    description: The number of questions to generate
+
+outputs:
+  questions:
+    type: list
+
+sample:
+  persona: a confused customer, not proficient in web interfaces
+  context: contoso outdoors shopping website
+  product: hiking boots
+  count: 5
+  questions: >
+    {"question":"How do I find the right size of hiking boots for my feet on this website?"}
+    {"question":"Are there instructions available for how to measure my foot for the correct boot fit?"}
+    {"question":"Can someone guide me through the process of selecting a pair of hiking boots?"}
+    {"question":"I'm not sure what the different boot terms mean, is there a glossary or help section?"}
+    {"question":"What is the return policy in case the hiking boots don't fit well when I receive them?"}
```

Comment:
this should look something like this:
```yaml:
questions:
   - How do I find the right size of hiking boots for my feet on this website?
   - Are there instructions available for how to measure my foot for the correct boot fit?
```
In general things should be yaml formatted (unless folks define things externally in files)

Reply:
in this case, I'm asking the model to output the results as a JSONL string, so this {} structure is what the model actually outputs

Reply:
modified the prompts to move example i/o into the description